core/src/muxing.rs

```diff
@@ -60,46 +60,35 @@ pub trait StreamMuxer {
     ///
     /// May panic or produce an undefined result if an earlier polling of the same substream
     /// returned `Ready` or `Err`.
-    fn poll_outbound(
-        &self,
-        substream: &mut Self::OutboundSubstream,
-    ) -> Poll<Option<Self::Substream>, IoError>;
+    fn poll_outbound(&self, s: &mut Self::OutboundSubstream) -> Poll<Option<Self::Substream>, IoError>;
 
     /// Destroys an outbound substream. Use this after the outbound substream has finished, or if
     /// you want to interrupt it.
-    fn destroy_outbound(&self, substream: Self::OutboundSubstream);
+    fn destroy_outbound(&self, s: Self::OutboundSubstream);
 
     /// Reads data from a substream. The behaviour is the same as `std::io::Read::read`.
     ///
     /// If `WouldBlock` is returned, then the current task will be notified once the substream
     /// is ready to be read, similar to the API of `AsyncRead`.
     /// However, for each individual substream, only the latest task that was used to call this
     /// method may be notified.
-    fn read_substream(
-        &self,
-        substream: &mut Self::Substream,
-        buf: &mut [u8],
-    ) -> Result<usize, IoError>;
+    fn read_substream(&self, s: &mut Self::Substream, buf: &mut [u8]) -> Result<usize, IoError>;
```

Comment:
Why rename `substream` to `s`? This is less descriptive.

Reply:
Given that it has `Substream` in its type right next to it and does not use the variable name since there is no method body I felt that having the method declaration on a single line is better.